client/settings/account-details/index.js

```diff
@@ -3,50 +3,69 @@ import { createInterpolateElement } from '@wordpress/element';
 import React from 'react';
 import SectionStatus from '../../components/section-status';
 import './style.scss';
+import { useAccount, useGetCapabilities } from 'wcstripe/data/account';
+
+const useIsCardPaymentsEnabled = () =>
+	useGetCapabilities().card_payments === 'active';
+
+const useAreDepositsEnabled = () => {
+	const { data } = useAccount();
+
+	return (
+		data.account?.payouts_enabled &&
```

Comment:
Can we avoid having to specify `data`?

Reply:
`data` is the name of the information returned by the `WC_REST_Stripe_Account_Controller`.
The `WC_REST_Stripe_Account_Controller` returns an `account` attribute.

We could rename `data` to `account`, but then we'd have to do `account.account`.

The reason why we have an `account` attribute returned by `WC_REST_Stripe_Account_Controller` is that with that endpoint we're _probably_ also going to add the `capabilities` attribute, returned by the `/v1/accounts/{account_id}/capabilities` endpoint in the Stripe API: https://stripe.com/docs/api/capabilities

Reply:
Gotcha, but with settings you get something like:

```js
{
  data: {
    foo: 'actual settings go here',
  },
  isLoading: false,
  someOtherMetaField: 'quux',
}
```

And when you try to use the settings, you get the values from `data` rather than the entire state. Maybe I misunderstand something but couldn't we be returning the equivalent of settings' `data` subtree here? (Unless we're already doing that.)

Reply:
Ah, got it! `useAccount()` uses the `getAccountData()` selector, which returns `state.account.data` (i.e.: it already returns the subtree)

Reply:
Thanks for the explanation! 🙌